### ISSUE TITLE
Nord light colorscheme has poor contrast on hovered items in the shell

### ISSUE THREAD

**Issue by ludvigng** [vinceliuice/Graphite-gtk-theme#55]
Hello!

Seems like the nord light version of the shell theme has poor contrast on hovered items. Looks fine once in the submenus though:

![Skärmbild från 2022-04-12 12-10-18](https://user-images.githubusercontent.com/50339757/162937861-d6f0c377-c09d-4a72-a980-ca3b7b5c4c8b.png)![Skärmbild från 2022-04-12 12-10-36](https://user-images.githubusercontent.com/50339757/162937872-cba5d1e2-084d-4353-818c-ea72aa8b9094.png)

**Comment by icyrockcom** [comment]
Agreed, for nord standard the contrast is poor in submenus as well:

![image](https://user-images.githubusercontent.com/785140/218375706-f0e76414-aa76-4a0b-8e78-3cd098972e56.png)

I *think* this is a relatively new issue though. I think it only started happening on my end a few months back.

**Comment by vinceliuice** [comment]
What's your system?

**Comment by icyrockcom** [comment]
I'm on NixOS, tracking the unstable channel. This is the definition if you're familiar with nix:

https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/data/themes/graphite-gtk-theme/default.nix#L30

It seems to have the latest released version.

**Comment by icyrockcom** [comment]
Interesting, this seems to be working now. I know I've updated Gnome 43 -> 44, not sure if there were other items that were changed that could have fixed this.

![image](https://user-images.githubusercontent.com/785140/236298984-e79de542-cfe1-4984-9aad-3c978638fdd4.png)
